### PR TITLE
[Docs] delete unused cfg value `lora_out_dir`

### PR DESCRIPTION
**Pull request by hamelsmu**
`lora_out_dir` is not an active config value `output_dir` is the config value to use.  I confirmed this with @winglian 

I also tested this and it doesn't appear that this config value does anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -643,7 +643,8 @@ max_memory:
 # If you want to use 'lora' or 'qlora' or leave blank to train all parameters in original model
 adapter: lora
 # If you already have a lora model trained that you want to load, put that here.
-# This means after training, if you want to test the model, you should set this to the value of `lora_out_dir`.
+# This means after training, if you want to test the model, you should set this to the value of `output_dir`.
+# Note that if you merge an adapter to the base model, a new subdirectory `merged` will be created under the original directory, which you will want to use instead.
```
Comment (NanoCode012) [inline README.md]:
```suggestion
# Note that if you merge an adapter to the base model, a new subdirectory `merged` will be created under the `output_dir`.
```

Comment (NanoCode012) [reply README.md]: Please also add info to find the merged model within `{output_dir}/merged` folder in the `### Merge LORA to base` section for visibility. I had some people ask where to find the final model.

**Review by NanoCode012 — COMMENTED** [review]
Nice catch!

File: README.md (hunk 1 above)
Comment (hamelsmu) [reply README.md]: @NanoCode012 I have some questions about that section (`### Merge LORA ...`).  I'll message you on discord

**Review by winglian — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hamelsmu) [reply README.md]: I'll merge this and then work on that section after I confirm it works like I think it does

**Pull request merged**